packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -277,4 +279,63 @@ describe("TinyliciousClient", () => {
         await corruptedAliasOp(runtimeOf(do1), "alias");
         assert(await dataCorruption);
     });
+
+    /**
+     * Scenario: Test if TinyliciousClient with forceWriteMode set to true forces the container to write mode.
+     * Keep the original behaviour otherwise.
+     *
+     * Expected behavior: Setting forceWriteMode to true in TinyliciousClientProps should force the container's
+     * connectionMode to `write` while keeping the original behavior if forceWriteMode is not set to true.
+     */
+    it("forces the container to connect with write mode when forceWriteMode is set to true ", async () => {
+        const getConnectionModeFromTinyliciousClient = async (
+            props?: TinyliciousClientProps,
+        ): Promise<ConnectionMode> => {
+            const client = new TinyliciousClient(props);
+            const { container } = await client.createContainer(schema);
+            await container.attach();
+
+            const map1 = container.initialObjects.map1 as SharedMap;
+            map1.set("newpair-id", "test");
```

Comment:
Attempting to write to a map while in read mode generally forces a reconnect in write mode -- I think these tests are expected to pass without writing to a map?

Reply:
These tests pass even when I remove this, I have removed writing to a map